src/infrastructure/figma/transformers/figma-file-transformer.ts

```diff
@@ -39,7 +40,7 @@ export const transformFileToAtlassianDesign = ({
 		status: AtlassianDesignStatus.NONE,
 		type: AtlassianDesignType.FILE,
 		lastUpdated: fileResponse.lastModified,
-		lastUpdatedBy: { id: fileMetaResponse.file.last_touched_by.id },
+		...(lastUpdatedBy ? { lastUpdatedBy: { id: lastUpdatedBy.id } } : {}),
```

Comment:
**suggestion:** Please, see https://github.com/atlassian-labs/figma-for-jira/pull/231/files#r1593215557.